src/drive/web/modules/filelist/generateFileUrl.js

```diff
@@ -0,0 +1,27 @@
+import { generateWebLink } from 'cozy-client'
+
+export const generateFileUrl = ({ file, client, isFlatDomain }) => {
```

Comment:
Maybe it should be `generateShortcutUrl ` since it only handle that kind of files?